docs/build/puppeteer-scraper-tutorial.md

```diff
@@ -71,7 +71,8 @@ And as we already know, there's only one.
 
 ```js
 // Using Puppeteer
-const title = await page.$eval('header h1', (el => el.textContent));
+const title = await page
+    .$eval('header h1', (el => el.textContent));
```

Comment:
Same principle here. `await page` looks like a broken piece of code.

Reply:
Here, could I do something like this:
```
const title = await page.$eval(
    'header h1', (el => el.textContent)
);
```
instead?